hw/ip/usbdev/dv/README.md

```diff
@@ -62,10 +70,27 @@ This does basic USB device initialization and is only disabled for `usbdev_commo
 
 Covergroups for functional coverage (as collected by `usbdev_env_cov`) are listed in the testplan at `hw/ip/usbdev/data/usbdev_testplan.hjson`.
 
+The following covergroups are implemented to give confidence that the DUT has been adequately stimulated during simulation:
+
+- common covergroup for interrupts hw/dv/sv/cip_lib/cip_base_env_cov.sv: Cover interrupt value, interrupt enable, intr_test, interrupt pin
+- pids_cg to ensure that the DUT has encountered all of the Packet Identifiers that it is expected to handle as a Full Speed USB device.
```

Comment:
Probably worth using some back-ticks to pick out the symbol names in this list. For example, `pids_cg` on this line.

Reply:
Okay, done. Also added another covergroup, because I think it's worth checking the PIDs in each direction; ie. all types of packets that are expected to be transmitted or received.